packages/utilities/psammead-assets/src/svgs.stories.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import styled from 'styled-components';
+import { node, number, shape } from 'prop-types';
+import { storiesOf } from '@storybook/react';
+import { C_POSTBOX } from '@bbc/psammead-styles/colours';
+import { number as numberKnob, withKnobs } from '@storybook/addon-knobs';
+import notes from '../README.md';
+import { news, igbo, yoruba, thai, pidgin } from './svgs';
+
+const Svg = styled.svg`
+  display: block;
+  fill: #fff;
+`;
+
+const Container = styled.div`
+  background: ${C_POSTBOX};
```

Comment:
Talking with @jamesbrumpton when he was testing this, we discussed the purpose of these SVG Storybook stories, which are distinct from the Brand Storybook stories, and the purpose of them.

By having these SVG stories have the brand colours, it can be confusing for those not familiar with the code to distinguish the difference between the usage & reusability of SVGs in this file.

As a result, I'd strongly recommend updating the background colour in this file to be a colour that isn't a brand colour. This is so that non-engineers who look at our Storybook examples can clearly see that the SVGs are independent of any specific branding colour and indicate the reusability of these SVGs.

```suggestion
  background-color: black;
```

I understand that since this ticket is needed as a prerequisite for other work, you might prefer to have this merged and open another PR to just update the colour on this Story. I'd be happy with that option.